### DISCUSSION TITLE
Is it possible to select text on the slide when in presenter view?

### DISCUSSION THREAD

**Discussion by hakib** [marp-team/marp#108] [Q&A] [answered]
The presenter view is very useful. Mainly the ability to see the next slide. 

Nowadays, many talks are recorded or broadcasted, and I personally find it useful to mark some text with my cursor. However, when in presenter view it is not possible to select text with your cursor. I tried forcing `user-select: text | all` in several places but it doesn't seem to have any effect.

**Is there a way I can select text on a slide with my cursor when in presenter view?**

I don't mind overriding some CSS manually just for the recording, I just need someone to point me in the right direction.

**Comment by yhatt** [comment, marked as answer]
No. [Marp CLI's `bespoke` template is disabling any kind of interaction for the slide in the presenter view](https://github.com/marp-team/marp-cli/blob/c7c2f83e2c02543678dec5821ff9950694c0f8c1/src/templates/bespoke/bespoke.scss#L212). The reason is to avoid misleading that all kind of the state for the presenter view will be synchronized with the main view.

When enabled some interactions such as text selection and the hyperlink, a speaker who is using the presenter view will certainly expect state that is synchronized with the main view. However, Marp CLI's `bespoke` template is simply providing sync of the current page, and not to sync the selection state.

If the speaker had used the text selection as marker without noticed this fact, **listeners who are watching the main view cannot see the emphasized selection**. The same may be said for the interactive element, such as `<video>` and `<input>`.

To prevent a trouble like that, the presenter view provided by `bespoke` template cannot interact with the slide.

**Reply by hakib** [reply]
This makes sense. Thank you for taking the time to write such detailed response.
